examples/server/server.c

```diff
@@ -240,63 +251,6 @@ int ServerEchoData(SSL* ssl, int clientfd, int echoData, int throughput)
     return EXIT_SUCCESS;
 }
 
-static void ServerRead(WOLFSSL* ssl, char* input, int inputLen)
-{
-    int ret, err;
-    char buffer[CYASSL_MAX_ERROR_SZ];
-
-    /* Read data */
-    do {
-        err = 0; /* reset error */
-        ret = SSL_read(ssl, input, inputLen);
-        if (ret < 0) {
-            err = SSL_get_error(ssl, 0);
-
-        #ifdef WOLFSSL_ASYNC_CRYPT
-            if (err == WC_PENDING_E) {
-                ret = wolfSSL_AsyncPoll(ssl, WOLF_POLL_FLAG_CHECK_HW);
-                if (ret < 0) break;
-            }
-            else
-        #endif
-            if (err != SSL_ERROR_WANT_READ) {
-                printf("SSL_read input error %d, %s\n", err,
-                                                 ERR_error_string(err, buffer));
-                err_sys("SSL_read failed");
-            }
-        }
-    } while (err == WC_PENDING_E);
-    if (ret > 0) {
-        input[ret] = 0; /* null terminate message */
-        printf("Client message: %s\n", input);
-    }
-}
-
-static void ServerWrite(WOLFSSL* ssl, const char* output, int outputLen)
-{
-    int ret, err;
-    char buffer[CYASSL_MAX_ERROR_SZ];
-
-    do {
-        err = 0; /* reset error */
-        ret = SSL_write(ssl, output, outputLen);
-        if (ret <= 0) {
-            err = SSL_get_error(ssl, 0);
-
-        #ifdef WOLFSSL_ASYNC_CRYPT
-            if (err == WC_PENDING_E) {
-                ret = wolfSSL_AsyncPoll(ssl, WOLF_POLL_FLAG_CHECK_HW);
-                if (ret < 0) break;
-            }
-        #endif
-        }
-    } while (err == WC_PENDING_E || err == SSL_ERROR_WANT_WRITE);
-    if (ret != outputLen) {
-        printf("SSL_write msg error %d, %s\n", err,
-                                                 ERR_error_string(err, buffer));
-        err_sys("SSL_write failed");
-    }
-}
```

Comment:
Hi Kincade, it looks like a bunch of TLS 1.3 changes are removed here. Perhaps a bad merge?